chainer/training/triggers/irregular_trigger.py

```diff
@@ -0,0 +1,50 @@
+class IrregularTrigger(object):
+
+    """Trigger invoked when specific value reaches the indicated values.
```

Comment:
Currently this trigger can specify only `iteration` or `epoch`.  So 
```Trigger invoked at specified iteration or epoch.```
is better to understand.